warehouse/query-core/src/main/java/datawave/query/jexl/visitors/IteratorBuildingVisitor.java

```diff
@@ -236,7 +234,32 @@ public Object visit(ASTJexlScript node, Object data) {
     @Override
     public Object visit(ASTAndNode and, Object data) {
         QueryPropertyMarker.Instance instance = QueryPropertyMarker.findInstance(and);
-        if (instance.isType(ExceededOrThresholdMarkerJexlNode.class)) {
+        if (instance.isType(ExceededTermThresholdMarkerJexlNode.class)) {
```

Comment:
will the `ExceededTermThresholdMarkerJexlNode` class need to be handled differently by the `BoundedRangeIndexExpansionVisitor` and other usages such as `RangeStreamInfo`, `EntryParser` now? or it is fine since it is still delayed